app/controllers/SingleRecordController.scala

```diff
@@ -74,9 +74,14 @@ class SingleRecordController @Inject() (
           )
         )
 
+        val categoryValue         = record.category.toString match {
+          case "1" => "Category 1"
```

Comment:
does it needs to be string match? looks like they are integers

Reply:
thanks 😄 updated now